### PR TITLE
Branch final junit

### LINKED ISSUE #153: Do JUnit for critical methods in Income/Expense

### LINKED ISSUE #154: JavaDoc for Income/Expense

### PR DESCRIPTION
**Pull request by NeoMinWei**
fixes #153 
fixes #154

### CODE REVIEW

File: src/test/java/seedu/financialplanner/commands/AddCashflowCommandTest.java
Comment (ryan1604) [inline src/test/java/seedu/financialplanner/commands/AddCashflowCommandTest.java:1]: Very extensive testing 👍

**Review by ryan1604 — APPROVED** [review]
LGTM

File: src/test/java/seedu/financialplanner/commands/AddCashflowCommandTest.java [hunk 1]
```diff
@@ -1,35 +1,141 @@
...
+            assertEquals("Description cannot be left empty.", e.getMessage());
+        }
+        try {
+            AddCashflowCommand testEntry = new AddCashflowCommand(Parser.
+                    parseRawCommand("add income /a 1 /t salary /r 1 /d 1  /x"));
+            fail();
+        } catch (IllegalArgumentException e) {
+            assertEquals("Unknown extra argument: x", e.getMessage());
+        }
+    }
```
Comment (wwweert123) [inline src/test/java/seedu/financialplanner/commands/AddCashflowCommandTest.java:98]: Wow upz

File: text-ui-test/input.txt [hunk 2]
```diff
@@ -2,6 +2,10 @@ add income /a 5000 /t salary /d part time job
 add income /a 123.12 /t allowance
 add expense /a 100 /t entertainment /d netflix
 add expense /a 123.21 /t shopping
+list
+list income
+list expense
+list recurring
```
Comment (wwweert123) [inline text-ui-test/input.txt:8]: Nice testing

**Review by wwweert123 — APPROVED** [review]
Didnt really read but looks good!

**Pull request merged**